dockerfile_parse/__init__.py

```diff
@@ -10,6 +10,7 @@
 # This file is generated from template: rel-eng/version_init.template
 #
 
+from __future__ import absolute_import
 from .parser import DockerfileParser
```

Comment:
The comment says this file is generated from template, maybe the import should be added to the template instead? Or just silence the warning, since the import is relative anyway

Reply:
Yes, add it into template too

Reply:
Thanks, I didn't get attention for that :(